mod-audit-server/src/main/java/org/folio/services/acquisition/impl/OrderAuditEventsServiceImpl.java

```diff
@@ -1,37 +1,37 @@
 package org.folio.services.acquisition.impl;
 
+import static org.folio.util.ErrorUtils.handleFailures;
+
 import io.vertx.core.Future;
-import io.vertx.pgclient.PgException;
 import io.vertx.sqlclient.Row;
 import io.vertx.sqlclient.RowSet;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.folio.dao.acquisition.OrderEventsDao;
-import org.folio.kafka.exception.DuplicateEventException;
 import org.folio.rest.jaxrs.model.OrderAuditEvent;
 import org.folio.rest.jaxrs.model.OrderAuditEventCollection;
 import org.folio.services.acquisition.OrderAuditEventsService;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service
 public class OrderAuditEventsServiceImpl implements OrderAuditEventsService {
 
   private static final Logger LOGGER = LogManager.getLogger();
 
-  public static final String UNIQUE_CONSTRAINT_VIOLATION_CODE = "23505";
-
-  private OrderEventsDao orderEventsDao;
+  private final OrderEventsDao orderEventsDao;
 
-  @Autowired
   public OrderAuditEventsServiceImpl(OrderEventsDao orderEvenDao) {
     this.orderEventsDao = orderEvenDao;
   }
 
   @Override
   public Future<RowSet<Row>> saveOrderAuditEvent(OrderAuditEvent orderAuditEvent, String tenantId) {
     LOGGER.debug("saveOrderAuditEvent:: Saving order audit event with Id={} for tenantId={}", orderAuditEvent.getId(), tenantId);
-    return orderEventsDao.save(orderAuditEvent, tenantId).recover(throwable -> handleFailures(throwable, orderAuditEvent.getId()));
+    return orderEventsDao.save(orderAuditEvent, tenantId)
+      .recover(throwable -> {
+        LOGGER.error("handleFailures:: Handling Failures with Id : {}", orderAuditEvent.getOrderId());
```

Comment:
It is no clear what is "Handling Failures with Id" means, lets rename to something like that: "Could not save order audit event with Id: {} for Order id: {} 
And the same in other places

Reply:
and also add throwable to the log

Reply:
![image](https://github.com/folio-org/mod-audit/assets/113523904/9b8e13d3-03b6-41bf-8bcd-cdcae6cc99ff)
with this type of message, error is already exists, so changing to your suggestion message may be duplicate

Reply:
we can change to another message

Reply:
basically this catch block is needed here only to catch situation if LocalDateTime.ofInstant or JsonObject.mapFrom will thow an exception. In case if for example DB will be unaccesiblle - we will have just failed promise, but no exception will be handled in this catch block, so we can remain 2 the same messages

Reply:
Thanks to explanation. is Could not save order audit event for Order id: {} in tenantId: {} message okay because we are also giving information about tenantId